Terminal.Gui/Views/ComboBox.cs

```diff
@@ -176,6 +176,21 @@ private void Initialize ()
 			}
 		}
 
+		/// <summary>
+		///If set to true its not allow any changes in the text.
+		/// </summary>
+		public bool ReadOnly {
+			get => search.ReadOnly;
+			set {
+				search.ReadOnly = value;
+				if (search.ReadOnly) {
+					if (search.ColorScheme != null) {
+						search.ColorScheme.Normal = search.ColorScheme.Focus;
```

Comment:
Good point, I really didn't worried about this, because if we use a ReadOnly combo box is for the app life. But this can be discusible and I can saving the original value e restore back if ReadOnly is set to false. I'm ready for suggestions.

Reply:
Since the `ReadOnly` is set at a constructor initialization, the `ColorScheme` may be null and is useless saved him to restore a null object. So, if it is needed to set the `ReadOnly` to true the `ColorScheme` must also be set.